operations/mimir-tests/test-multi-zone-with-store-gateway-automated-downscaling.jsonnet

```diff
@@ -0,0 +1,45 @@
+local mimir = import 'mimir/mimir.libsonnet';
+
+mimir {
+  _config+:: {
+    namespace: 'default',
+    external_url: 'http://test',
+
+    storage_backend: 'gcs',
+    blocks_storage_bucket_name: 'blocks-bucket',
+
+    ruler_enabled: true,
+    ruler_storage_bucket_name: 'rules-bucket',
+
+    alertmanager_enabled: true,
+    alertmanager_storage_bucket_name: 'alerts-bucket',
+
+    multi_zone_ingester_enabled: true,
+    multi_zone_ingester_replicas: 3,
+
+    multi_zone_store_gateway_enabled: true,
+    multi_zone_store_gateway_replicas: 3,
+    store_gateway_automated_downscale_enabled: true,
```

Comment:
When I left the comment about this test, I completely missed the fact that we're enabling it by default. Then this test doesn't make sense. Please remove it.

Reply:
I have renamed the test into `test-multi-zone-without-store-gateway-automated-downscaling.jsonnet` and tested what happen when we disable automated downscaling.